### PR TITLE
feat(Badge): add navy stroke variant

### PR DESCRIPTION
**Pull request by jcenglish**
### Overview

Add navy stroke option. This is specifically for the new pricing page design where the available gray stroke option does not meet accessible contrast standards against pale yellow and yellow backgrounds.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/Ym03UHdFa0ctNFVoMndKze/Plan-Split%3A-pricing-and-payment-v2?node-id=949%3A71671
- [x] Related to JIRA ticket: GRW-194
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.165-alpha.6b20ae.0<br/>@codecademy/gamut-labs@33.0.1-alpha.6b20ae.0<br/>@codecademy/gamut@48.3.1-alpha.6b20ae.0<br/>@codecademy/monaco@0.6.60-alpha.6b20ae.0<br/>@codecademy/styleguide@59.0.1-alpha.6b20ae.0<br/></pre>

File: packages/gamut/src/Badge/index.tsx [hunk 1]
```diff
@@ -34,6 +34,12 @@ const colorVariants = variant({
       color: 'text-disabled',
       bg: 'transparent',
     },
+    strokeNavy: {
+      border: 1,
+      borderColor: 'navy',
+      color: 'navy',
+      bg: 'transparent',
+    },
```
Comment (dreamwasp) [inline packages/gamut/src/Badge/index.tsx:42]: i think probably want this to be responsive to ColorMode - maybe a name like `strokeContrast` + you can just replace `navy` here with the appropriate [ColorMode](https://gamut.codecademy.com/?path=/docs/foundations-colormode--page) semantic color equivalent

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62dff52f294be400b96bb4a0--gamut-preview.netlify.app>https://62dff52f294be400b96bb4a0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62dff52f294be400b96bb4a0>Deploy Logs</a></p>

File: packages/gamut/src/Badge/index.tsx (hunk 1 above)
Comment (jcenglish) [reply packages/gamut/src/Badge/index.tsx]: How's this?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.165-alpha.da821d.0<br/>@codecademy/gamut-labs@33.0.1-alpha.da821d.0<br/>@codecademy/gamut@48.3.1-alpha.da821d.0<br/>@codecademy/monaco@0.6.60-alpha.da821d.0<br/>@codecademy/styleguide@59.0.1-alpha.da821d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62e009dd6a4f1614af85aa6d--gamut-preview.netlify.app>https://62e009dd6a4f1614af85aa6d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62e009dd6a4f1614af85aa6d>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
🔥

**Pull request merged**